trajectory_publisher/include/trajectory_publisher/trajectoryPublisher.h

```diff
@@ -53,6 +57,7 @@ class trajectoryPublisher
   Eigen::Vector3d p_targ, v_targ, a_targ;
   Eigen::Vector3d p_mav_, v_mav_;
   Eigen::Vector3d shape_origin_, shape_axis_;
+	Eigen::Vector3d p_home_;
```

Comment:
The spacing format seems to be different. Please align the spaces with the rest of the code